examples/opentelemetry-otlp.rs

```diff
@@ -41,11 +41,13 @@ fn init_meter_provider() -> SdkMeterProvider {
 
     // For debugging in development
     let stdout_reader = PeriodicReader::builder(
-        opentelemetry_stdout::MetricsExporter::default(),
+        opentelemetry_stdout::MetricExporter::default(),
         runtime::Tokio,
     )
     .build();
 
+    // NOTE: Views are gated behind opentelemetry's `spec_unstable_metrics_views` feature.
```

Comment:
Thanks for digging into this, but while views are unstable I think we should avoid showing them off in our examples.

Reply:
Done!